src/types.mligo

```diff
@@ -51,10 +51,9 @@ type nonce = nat
 // Represents whether a voter has voted against (false) or for (true) a given proposal.
 type vote_type = bool
 
-type vote_status =
+type staked_vote =
  { upvotes : nat
  ; downvotes: nat
```

Comment:
Do we need the separate `upvotes`, `downvotes` fields now?

Reply:
I guess we don't need to separate them anymore, but it may still be useful for the test (Easier to do the check).

I combine them anyway, you can check it now.

Reply:
LGTM now. But maybe @pasqu4le also better signoff this change as well.

Reply:
Yep, makes sense to me too.

This also was only used to keep historical data and is better off removed now :ok_hand: